app/src/main/java/org/fossasia/openevent/app/core/sponsor/create/CreateSponsorFragment.java

```diff
@@ -32,19 +34,38 @@
 
     private SponsorCreateLayoutBinding binding;
     private Validator validator;
+    private long sponsorId;
+    private boolean isUpdateSponsor;
 
     public static CreateSponsorFragment newInstance() {
         return new CreateSponsorFragment();
     }
 
+    public static CreateSponsorFragment newInstance(long sponsorId) {
+        Bundle bundle = new Bundle();
+        bundle.putLong("sponsor_id", sponsorId);
+        CreateSponsorFragment createSponsorFragment = new CreateSponsorFragment();
+        createSponsorFragment.setArguments(bundle);
```

Comment:
Add a static factory method in CreateSponsorFragment `newInstance` instead of manually doing this